database/things/co-dfns.pldb

```diff
@@ -1,10 +1,10 @@
 title co-dfns
 appeared 2012
 type pl
-originCommunity https://github.com/Co-dfns/
-country Unknown
 website https://www.patreon.com/arcfide
 reference https://dl.acm.org/doi/10.1145/2627373.2627384
+country Unknown
```

Comment:
Aaron Hsu, the main researcher, is based in the United States.
https://confengine.com/user/arcfide

Reply:
Fixed in latest commit.